project.yaml

```diff
@@ -10,3 +10,18 @@ actions:
     outputs:
       highly_sensitive:
         cohort: output/input_main.csv
+
+  generate_study_population:
+    run: cohortextractor:latest generate_cohort --study-definition study_definition --index-date-range "2021-01-01 to today by month" --skip-existing --output-dir=output/measures --output-format=csv
```

Comment:
We're moving towards recommending feather instead of CSVs for the input file. It's fine to keep it as a csv while you are working on it though.